circle.yml

```diff
@@ -43,6 +43,18 @@ test:
     - mkdir -p $CIRCLE_TEST_REPORTS/unit
     - npm run unit_coverage
     - npm run multiple_backend_test
+    # Run S3 with multiple data backends ; run ft_awssdk
```

Comment:
No, I moved the multiple backend tests up so they get run first so we can see if they fail right away.